docs/source/conf.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'EthPM Package Manifest Specification'
+copyright = u'2018, Piper Merriam, et al.'
+author = u'Piper Merriam, et al.'
+
+# The short X.Y version
+version = u''
```

Comment:
This is python2 unicode string syntax. We should probably be using python3 which doesn't have the `u` prefix.